### DISCUSSION TITLE
[static-fun] How to exporting a static site by subdomain

### DISCUSSION THREAD

**Discussion by flutterforsale** [vercel/next.js#10820] [Help]
Hi, I want apply feature static exporting for static-fun project (https://github.com/zeit/static-fun)

Ex: I have a add a button "Deploy Page"  in static-fun backend

PageList:

1. subdomain1.static.fun  |  Deploy Page
1. subdomain2.static.fun |  Deploy Page
1. subdomain3.static.fun |  Deploy Page

When click DeployPage button on subdomain1.static.fun system will run a service to export static for subdomain1

When click DeployPage button on subdomain2.static.fun system will run a service to export static for subdomain2

After, when user open subdomain1.static.fun on browse. if it static file exits page will load from this. else page will get page info from database.

How i cant do this. Thank you so much!

**Comment by flutterforsale** [comment]
@waptik can you help me.

**Reply by waptik** [reply]
I don't have much knowledge about it but these might help: #10699 , #9524

**Comment by flutterforsale** [comment]
Thanks @waptik I am also looking for a solution
